From bd379fd6849391cd44415505e738a15c3c1004ac Mon Sep 17 00:00:00 2001
From: pet <pet@cow.(none)>
Date: Wed, 2 Dec 2009 11:11:50 +0800
Subject: out test

---
 git.muse |    6 +++++-
 1 files changed, 5 insertions(+), 1 deletions(-)

diff --git a/git.muse b/git.muse
index f7a56b4..51305fd 100644
--- a/git.muse
+++ b/git.muse
@@ -6,7 +6,11 @@
 
 [[git_command]]
 
-** track rename history
+** track one flile  history
 <example>
 pet@cow:~/edocs$ git log --follow auth_security.muse
+pet@cow:~/edocs$ git show 9188a9bdba8d5a3f94be6e1a1bc634b58b398b6e git.muse
 </example>
+
+git format-patch --keep-subject -o out origin
+
-- 
1.6.0.4

